includes/Core/Assets/Stylesheet.php

```diff
@@ -61,12 +61,21 @@ public function register( Context $context ) {
 		if ( $this->args['fallback'] && wp_style_is( $this->handle, 'registered' ) ) {
 			return;
 		}
+		$src     = $this->args['src'];
+		$version = $this->args['version'];
+
+		$filename = Manifest::get_filename( $this->handle );
+
+		if ( ! is_null( $filename ) ) {
```

Comment:
We can relax this condition a bit. If an empty string were returned for example, that shouldn't trigger the same behavior as if there was a filename.

```suggestion
		if ( $filename ) {
```

Reply:
https://github.com/google/site-kit-wp/pull/4095/commits/01d9abe1d5ed557ff063b7758cb9cd8ac1bc1fd1